docs/configuration/permissions.md

```diff
@@ -8,10 +8,10 @@ The Bolt Permissions System
 Bolt uses a [Role-Based Permission](https://en.wikipedia.org/wiki/Role-based_access_control)
 system. This means that:
 
-* Every *user* can be a member of zero or more *roles*
+* Every *user* can have zero or more *roles*
```

Comment:
I think "can have" can be replaced with "has". It's not optional to have 'zero or more roles'
```suggestion
* Every *user* has zero or more *roles*
```